src/storage/redis_metadata.cc

```diff
@@ -378,7 +379,7 @@ rocksdb::Status StreamMetadata::Decode(const std::string &bytes) {
   GetFixed64(&input, &version);
   GetFixedCommon(&input, &size);
 
-  if (input.size() < 88) {
+  if (input.size() < 96) {
```

Comment:
It seems this line will break the compatibility of kvrocks in stream datatype.

Reply:
i.e., the old userdata of stream datatype will always not satisfy this condition.

Reply:
Oh, i don't recognize it. The variable `group_number` is added to `StreamMetadata` and it serves for quickly getting the consumer group number which will be used in `XINFO STREAM`. I can delete it if necessary and the number of consumer groups can be counted through iteration.

Reply:
I mean, you can add some new field to StreamMetadata, but you cannot make old data in user's storage inaccessible.

So if you want to add a new field, you need to consider to write a Decode function that can work well while facing old userdata without the field you just add.

Reply:
Got it. :)